java/src/jmri/jmrit/display/layoutEditor/BlockValueFile.java

```diff
@@ -41,10 +44,13 @@ public BlockValueFile() {
     private Document doc = null;
     private Element root = null;
 
-    /*
+    /**
      *  Reads Block values from a file in the user's preferences directory
      *  If the file containing block values does not exist this routine returns quietly.
      *  If a Block named in the file does not exist currently, that entry is quietly ignored.
+     * 
+     * @throws JDOMException
```

Comment:
Please explain why this throws these exceptions (it can be brief), so that when reading the documentation, I know that (for example), it throws the JDOMException if unable to parse the XML content and the IOException if unable to read file [what is the name of the file that is read?] in the user's data directory.

(Also FindBugs (I think, it may be another check on Jenkins) throws a warning if an ```@param``` or ```@throws``` or ```@return``` value in a Javadoc comment does not have a description.)

Reply:
Done.